### PR TITLE
Packages/py sphinxcontrib trio

### PR DESCRIPTION
**Pull request by Bidibulke**
Adding recipe for `sphinxcontrib-trio` (a python package)

### CODE REVIEW

File: lib/spack/docs/basic_usage.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+.. Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```
Comment (alalazo) [inline lib/spack/docs/basic_usage.rst]: Can you revert this change?

File: var/spack/repos/builtin/packages/py-sphinxcontrib-trio/package.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    homepage = "https://github.com/python-trio/sphinxcontrib-trio"
+    url = "https://files.pythonhosted.org/packages/ca/33/ee48d86e30bb3c5d72a47f49b1ebf5c23dd253b04d8d5fc3e6c68407a03e/sphinxcontrib-trio-1.1.2.tar.gz"
+
+    version('1.1.2', sha256='9f1ba9c1d5965b534e85258d8b677dd94e9b1a9a2e918b85ccd42590596b47c0')
+    version('1.1.0', sha256='d90f46d239ba0556e53d9a110989f98c9eb2cea76ab47937a1f39b62f63fe654')
+
+    depends_on('py-sphinx@1.7:')
+
+    def path(self):
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/py-sphinxcontrib-trio/package.py]: Is there a typo in the name of the function? Why you didn't use a patch directive instead?

**Review by alalazo — CHANGES_REQUESTED** [review]
A few questions. Also why the same patch has been committed under multiple file names?

File: lib/spack/docs/basic_usage.rst (hunk 1 above)
Comment (Bidibulke) [reply lib/spack/docs/basic_usage.rst]: Done.
I think I made a bad git command. Sorry

File: var/spack/repos/builtin/packages/py-sphinxcontrib-trio/package.py (hunk 2 above)
Comment (Bidibulke) [reply var/spack/repos/builtin/packages/py-sphinxcontrib-trio/package.py]: It's a typo and I didn't know the patch directive.
So I changed for the patch directive (more readable)

File: lib/spack/docs/basic_usage.rst (hunk 1 above)
Comment (alalazo) [reply lib/spack/docs/basic_usage.rst]: No worries, and thanks for your contributions!

**Review by alalazo — APPROVED** [review]
